stock_picking_line_sequence/models/stock.py

```diff
@@ -83,3 +85,25 @@ def button_validate(self):
         return super(
             StockPicking, self.with_context(keep_line_sequence=True)
         ).button_validate()
+
+    @api.model
+    def get_view(self, view_id=None, view_type="form", **options):
+        """Append the default sequence.
+        The context of `move_ids_without_package` is already overloaded
+        and replacing it in a view does not scale across other extension
+        modules.
+        """
+        res = super().get_view(view_id=view_id, view_type=view_type, **options)
+
+        if res.get("arch") and view_type == "form":
+            doc = etree.XML(res["arch"])
+            elements = doc.xpath("//field[@name='move_ids_without_package']")
+            if elements:
+                element = elements[0]
+                context = element.get("context", "{}")
+                context = "{}{}, {}".format(
```

Comment:
you could safe_eval it and then convert back to string.

Reply:
@oerp-odoo what exactly you suggest to safe_eval in this case?